src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from 'react'
+import { createRoot } from 'react-dom/client'
+import App from './App.tsx'
+import './scss/global.scss'
```

Comment:
please, use alias instead of relative path

Reply:
done